chain/client/src/client_actor.rs

```diff
@@ -1534,6 +1534,8 @@ impl ClientActor {
 impl Drop for ClientActor {
     fn drop(&mut self) {
         self.state_parts_client_arbiter.stop();
+        info!(target:"chain", "As ClientActor is dropped, initiate a graceful shutdown.");
+        System::current().stop();
```

Comment:
btw - will this still work after #6333 ?

Reply:
Yes, and it should fix the `pthread destroy mutex: Device or resource busy` issue.

Reply:
my point is, that it feels 'strange' that the ClientActor destructor is the thing that initiates 'global' shutdown.